# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/binutils/hotfix-gold-sparcv.patch
# Copyright (C) 2022 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- binutils-2.39/gold/configure.tgt.vanilla	2022-10-31 19:11:38.990566812 +0100
+++ binutils-2.39/gold/configure.tgt	2022-10-31 19:11:51.724565487 +0100
@@ -80,7 +80,7 @@
   targ_big_endian=false
   targ_extra_big_endian=true
   ;;
-sparc-*)
+sparc*-*)
   targ_obj=sparc
   targ_machine=EM_SPARC
   targ_size=32
